src/preact/base-element.js

```diff
@@ -341,16 +341,21 @@ function collectProps(Ctor, element, defaultProps) {
 
       const {single, name, props: slotProps = {}} = def;
 
-      // TBD: assign keys, reuse slots, etc.
+      // TBD: assign keys, etc.
```

Comment:
Nit: I still hope to reuse slots :)

Reply:
Ah, I had removed this because I modified this to reuse rather than overwrite existing `slot` attributes, but I can restore the comment if it means to do something different.